tools/codegen/gen_lazy_tensor.py

```diff
@@ -222,9 +223,13 @@ def gen_key(func: FunctionSchema) -> Tuple[str, str]:
                                        grouped_native_functions, backend_key, autograd_key)
 
     # Generate Dispatcher registrations which hook up the nativefunctions
+    internal_headers = [
+        "torch/csrc/lazy/generated/LazyNativeFunctions.h",
```

Comment:
@wconstab I think a similar change to what was discussed below with the `output_dir` needs to be applied here too, based on my testing